google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkPanelTest.java

```diff
@@ -34,11 +36,18 @@
 
   private CloudSdkService cloudSdkService;
 
-  private JLabel warningMessage;
+  private JTextPane warningMessage;
   private TextFieldWithBrowseButton cloudSdkDirectoryField;
 
-  private static final String INVALID_SDK_DIR_WARNING =
-      "No Cloud SDK was found in this directory.";
+  private static final String INVALID_SDK_DIR_WARNING = "<html>\n"
```

Comment:
I'd usually agree, but in this cause there are multiple warning messages that can be displayed in the same UI field (one for empty field, and one for invalid field) so this test checks that the right one is being shown.